components/branches/branches.html

```diff
@@ -8,8 +8,12 @@
   </button>
   <ul class="dropdown-menu pull-right" role="menu">
     <div class="option" onclick="event.stopPropagation()">
-      Only local branch?
-      <input type="checkbox" data-bind="checked: isLocalBranchOnly, "></input>
+      Remote
```

Comment:
A `<label>` around the text and the checkbox would be nice. Then you can click both the text and the checkbox to toggle it.